week02/analyze/Program.cs

```diff
@@ -16,23 +16,23 @@
 
 try {
     MysteryStack2.Run("5 0 /");
-    Console.WriteLine("WRONG: expected ApplicationException: Invalid Case 1!");
+    Console.WriteLine("WRONG: expected ApplicationException: Invalid Case 2!");
```

Comment:
Do we have the `Debug.WriteLine` versus `Console.WriteLine` issue here? Should we do something else here anyway that is more obvious? Maybe an assert or raise a different Exception? (I'm not sure on this. It might be fine as is, but I wanted to note it.)

Reply:
I didn't modify any code outside of the "code" sections each week to use automated unit tests, meaning that code like this in "analyze" will still build a binary and Console.WriteLine works fine. Mostly since the changes I was already making were so extensive I didn't further want to have to validate all learning content too.

 So I think since this didn't change otherwise than correcting the error of printing the wrong case, we can leave it as-is for now. If in the future we want to introduce real unit tests in other parts of the repo, the right thing to do would be to throw here so the test fails.